server/src/main/java/com/group10/lobcitydata/models/rapidapi/Birth.java

```diff
@@ -0,0 +1,37 @@
+package com.group10.lobcitydata.models.rapidapi;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Birth {
+    private String date;
+
+    private String country;
+
+    @JsonCreator
```

Comment:
No change necessary, but these are not necessary with the API we are consuming. I incorrectly added these, but for future ones we won't need the annotations.